server/storage/hot_region_storage_test.go

```diff
@@ -207,7 +198,7 @@ func (t *testHotRegionStorage) TestHotRegionDelete(c *C) {
 	num := 0
 	for next, err := iter.Next(); next != nil && err == nil; next, err = iter.Next() {
 		num++
-		c.Assert(reflect.DeepEqual(next, &historyHotRegions[defaultRemainDay-num]), IsTrue)
+		re.True(reflect.DeepEqual(next, &historyHotRegions[defaultRemainDay-num]))
```

Comment:
Equal is enough.

Reply:
I think "re.True" is more suitable for functions that return bool value.
And I see an example that has been merged.

```
- c.Assert(rep.GetReplicationStatus().GetDrAutoSync().GetPauseRegionSplit(), IsTrue)
+ re.True(rep.GetReplicationStatus().GetDrAutoSync().GetPauseRegionSplit())
```
https://github.com/tikv/pd/pull/5100/files#diff-f36769a34db4e8e3412376a3a7c5be18b2f7f8ab90287fc48c7f4445bc716218L221

Reply:
See https://github.com/tikv/pd/issues/5104, `require.Equal` is used to replace `reflect.DeepEqual`.

```suggestion
		re.Equal(&historyHotRegions[defaultRemainDay-num], next)
```

Reply:
Oh, I got it, I'll fix it.